packages/dds/map/src/mapKernel.ts

```diff
@@ -661,9 +678,11 @@ export class MapKernel {
                         0x2fd /* pendingMessageId does not match */);
                     this.submitMapClearMessage(op, localOpMetadata.previousMap);
                 },
-                getStashedOpLocalMetadata: (op: IMapClearOperation) => {
+                applyStashedOp: (op: IMapClearOperation) => {
+                    const copy = new Map<string, ILocalValue>(this.data);
```

Comment:
This is not necessary I think, since this is for local rollback which we don't expect/need for stashed ops? @scarlettjlee

Reply:
It won't be used in any of the currently supported scenarios, but from the perspective of the DDS API/functionality, it doesn't enforce that and has no reason to prevent it. It's possible that this could be used later for some other scenario.